api/order/orderRoute.js

```diff
@@ -1 +1,96 @@
-//order endpoints
+import express from 'express';
+import Order from './Order.js';
+import User from '../user/User.js';
+import Ticket from '../ticket/Ticket.js';
+
+const router = express.Router();
+
+router
+  .route('/')
+  .get(async (req, res) => {
+    const orders = await Order.find({});
+    res.status(200).json(orders);
+  })
+  .post(async (req, res) => {
+    const { userId, email, status, tickets } = req.body;
+    const user = await User.findById(userId);
+    if (user === undefined) {
+      res.status(400).json({
+        error: `Cannot find user with id: ${req.params.id}'`,
+      });
+      return;
+    }
+
+    const order = new Order({
+      user: userId,
+      email,
+      status,
+    });
+    await order.save();
+    tickets.forEach(async (ticket) => {
+      const newTicket = new Ticket({
+        screening: ticket.screeningId,
+        row: ticket.row,
+        column: ticket.column,
+        order: order.id,
+      });
+      await newTicket.save();
+      return ticket.id;
+    });
+    await order.save();
```

Comment:
duplicated line 29? Perhaps 40 to remain after tickets are incorporated in order